jib/jib-gradle.yaml

```diff
@@ -16,15 +16,49 @@ spec:
   steps:
   - name: build-and-push
     image: gcr.io/cloud-builders/gradle
+    command:
```

Comment:
This might indicate that we want a `jib-gradle` builder image that includes these files (and any future ones), without having to squeeze them into the build template.

If we go that route, my personal preference would be to base that image `FROM gradle` and not rely on the poorly-supported `gcr.io/cloud-builders/gradle` image. (Same with `maven`, tbh). That can happen in a later CL though if you'd prefer.